chains/container-chains/runtime-templates/frontier/src/xcm_config.rs

```diff
@@ -104,7 +103,7 @@ parameter_types! {
     pub RootLocation: Location = Location::here();
 
     // TODO: Revisit later
-    pub const ContainerToEthTransferFee: u128 = 2_700_000_000_000u128;
+    pub const ContainerToEthTransferFee: u128 = 3_500_000_000_000u128;
```

Comment:
Increased fee amount, because for this type of transfer the previous value is not enough.